tools/src/tester/types/eval.types.ts

```diff
@@ -87,16 +87,42 @@ export enum Result {
 export class OutputReference {
```

Comment:
I missed this when this was first added but this file is meant to hold type definitions only. This class should be in its own file.